spaceship/lib/spaceship/connect_api/models/user.rb

```diff
@@ -48,6 +56,12 @@ def self.find(client: nil, email: nil, includes: nil)
         client ||= Spaceship::ConnectAPI
         return all(client: client, filter: { email: email }, includes: includes)
       end
+
+      def fetch_visible_apps(client: nil, limit: nil)
```

Comment:
@joshdholtz Any thought on naming this kind of method with `get` vs `fetch`? 🤔

Reply:
@ainame This also made me think for a while and compare code in other models and I wasn't sure if the choice is correct. 🤔

Reply:
Me too honestly. Some uses "get" others do "fetch" 😅

Reply:
UGGHHHH... I hate naming 😅 I honestly don't know which it should be but I do want it to be consistent at some point 🤷‍♂️

Reply:
I've renamed it to `get` and it's used more in code.